script/lps22hb/main-lp.lua

```diff
@@ -0,0 +1,44 @@
+local SAMPLE_INTERVAL = 60000 -- 1 minute
+
+-- number of samples to bundle up locally before sending.
+-- if the device loses power before submitting these samples
+-- then they are lost forever. too many samples may reduce
+-- memory usage, require he.wait calls, and increases the
+-- amount of data you will lose.
+local SAMPLE_BUNDLE = 10
+
+local lps22hb = require('lps22hb')
+local sensor  = assert(lps22hb:new())
+
+local tbl = {}
+local idx = 1
+
+local now = he.now()
+while true do
+    local pressure = assert(sensor:read_pressure())
+    local temperature = assert(sensor:read_temperature())
+
+    tbl[idx] = {now, pressure, temperature}
+    print("stored temporary sample #"..idx)
```

Comment:
Nice work! Perhaps you could not use `..` here since string manipulation still happens on device, even though the result doesn't go anywhere. I've seen memory become pretty fragmented when using this extensively. 

Perhaps just use `,` instead of `..`?